packages/search/src/style.scss

```diff
@@ -227,6 +227,23 @@ $input-z-index: 20;
 
 			padding-left: 8px;
 		}
+
+		// Removes the fade overlay when the search is open and focused
+		&.has-focus {
+			.search-component__input-fade {
+
+				&::before {
+					display: none;
+				}
+
+				&.ltr {
+					/*!rtl:ignore*/
```

Comment:
Is this commentary still necessary?

Reply:
I guess. It's to let others know that we are not covering the `.rtl` case yet.

Reply:
Now the comment is not necessary anymore as we're handling the RTL case! We can remove it 🎉